roachtest: django failed

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2873147&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2873147&tab=artifacts#/django) on master @ [c40ed790b3a19f20de30375788b0359f25dace97](https://github.com/cockroachdb/cockroach/commits/c40ed790b3a19f20de30375788b0359f25dace97):


```
		--- FAIL: invalid_models_tests.test_relative_fields.AccessorClashTests.test_fk_to_fk (unexpected)
		--- FAIL: invalid_models_tests.test_models.OtherModelTests.test_m2m_unmanaged_shadow_models_not_checked (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexTogetherTests.test_pointing_to_fk (unexpected)
		--- FAIL: invalid_models_tests.test_models.OtherModelTests.test_ordering_pointing_to_two_related_model_field (unexpected)
		--- FAIL: sites_framework.tests.CurrentSiteManagerChecksTests.test_invalid_field_type (unexpected)
		--- FAIL: invalid_models_tests.test_models.UniqueTogetherTests.test_pointing_to_fk (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedQueryNameClashTests.test_hidden_fk_to_m2m (unexpected)
		--- FAIL: check_framework.test_model_checks.IndexNameTests.test_no_collision_abstract_model_interpolation (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.AccessorClashTests.test_m2m_to_integer (unexpected)
		--- FAIL: invalid_models_tests.test_models.JSONFieldTests.test_check_jsonfield (unexpected)
		--- FAIL: invalid_models_tests.test_models.JSONFieldTests.test_check_jsonfield_required_db_features (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexTogetherTests.test_list_containing_non_iterable (unexpected)
		--- FAIL: invalid_models_tests.test_models.UniqueTogetherTests.test_pointing_to_m2m (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ReverseQueryNameClashTests.test_fk_to_fk (unexpected)
		--- FAIL: invalid_models_tests.test_models.ConstraintsTests.test_check_constraints (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedNameClashTests.test_fk_to_m2m (unexpected)
		--- FAIL: check_framework.test_model_checks.DuplicateDBTableTests.test_no_collision_for_proxy_models (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexesTests.test_pointing_to_fk (unexpected)
		--- FAIL: invalid_models_tests.test_models.FieldNamesTests.test_pk (unexpected)
		--- FAIL: check_framework.test_model_checks.IndexNameTests.test_collision_across_apps (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexesTests.test_func_index_pointing_to_missing_field_nested (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedQueryNameClashTests.test_fk_to_integer (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.SelfReferentialFKClashTests.test_accessor_clash (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexesTests.test_index_include_pointing_to_missing_field (unexpected)
		--- FAIL: invalid_models_tests.test_models.OtherModelTests.test_ordering_pointing_to_related_model_pk (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexesTests.test_index_with_include (unexpected)
		--- FAIL: invalid_models_tests.test_deprecated_fields.DeprecatedFieldsTests.test_IPAddressField_deprecated (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.AccessorClashTests.test_fk_to_integer (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedQueryNameClashTests.test_hidden_fk_to_fk (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexesTests.test_pointing_to_missing_field (unexpected)
		--- FAIL: invalid_models_tests.test_models.OtherModelTests.test_ordering_pointing_to_missing_field (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexesTests.test_index_with_condition (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedQueryNameClashTests.test_hidden_fk_to_integer (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexTogetherTests.test_pointing_to_m2m_field (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedNameClashTests.test_m2m_to_fk (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexesTests.test_pointing_to_m2m_field (unexpected)
		--- FAIL: model_inheritance.test_abstract_inheritance.AbstractInheritanceTests.test_shadow_related_name_when_set_to_none (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexesTests.test_func_index_required_db_features (unexpected)
		--- FAIL: invalid_models_tests.test_models.ConstraintsTests.test_check_constraint_pointing_to_joined_fields_complex_check (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedNameClashTests.test_m2m_to_integer (unexpected)
		--- FAIL: invalid_models_tests.test_models.OtherModelTests.test_ordering_with_order_with_respect_to (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexesTests.test_index_with_include_required_db_features (unexpected)
		--- FAIL: invalid_models_tests.test_models.OtherModelTests.test_ordering_pointing_to_foreignkey_field (unexpected)
		--- FAIL: invalid_models_tests.test_models.OtherModelTests.test_two_m2m_through_same_relationship (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexesTests.test_index_with_condition_required_db_features (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.AccessorClashTests.test_m2m_to_m2m (unexpected)
		--- FAIL: invalid_models_tests.test_models.OtherModelTests.test_just_order_with_respect_to_no_errors (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_1) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63348 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.2]
- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2878583&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2878583&tab=artifacts#/django) on master @ [7495d19e50d8fd14140220f3ba5c7ad951abafa6](https://github.com/cockroachdb/cockroach/commits/7495d19e50d8fd14140220f3ba5c7ad951abafa6):


```
		--- FAIL: invalid_models_tests.test_models.ConstraintsTests.test_check_constraint_pointing_to_joined_fields (unexpected)
		--- FAIL: invalid_models_tests.test_models.UniqueTogetherTests.test_non_iterable (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedQueryNameClashTests.test_m2m_to_integer (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedNameClashTests.test_fk_to_fk (unexpected)
		--- FAIL: model_inheritance.test_abstract_inheritance.AbstractInheritanceTests.test_shadow_related_name_when_set_to_none (unexpected)
		--- FAIL: invalid_models_tests.test_models.FieldNamesTests.test_db_column_clash (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedQueryNameClashTests.test_m2m_to_fk (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedQueryNameClashTests.test_fk_to_integer (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedNameClashTests.test_m2m_to_integer (unexpected)
		--- FAIL: check_framework.test_model_checks.ConstraintNameTests.test_no_collision_across_apps_interpolation (unexpected)
		--- FAIL: invalid_models_tests.test_models.UniqueTogetherTests.test_non_list (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: invalid_models_tests.test_relative_fields.SelfReferentialFKClashTests.test_reverse_query_name_clash (unexpected)
		--- FAIL: invalid_models_tests.test_models.ConstraintsTests.test_check_constraint_pointing_to_m2m_field (unexpected)
		--- FAIL: invalid_models_tests.test_models.FieldNamesTests.test_ending_with_underscore (unexpected)
		--- FAIL: invalid_models_tests.test_models.FieldNamesTests.test_local_field_long_column_name (unexpected)
		--- FAIL: model_inheritance.test_abstract_inheritance.AbstractInheritanceTests.test_shadowed_fkey_id (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexTogetherTests.test_pointing_to_fk (unexpected)
		--- FAIL: invalid_models_tests.test_models.OtherModelTests.test_m2m_field_table_name_clash_database_routers_installed (unexpected)
		--- FAIL: invalid_models_tests.test_models.ConstraintsTests.test_unique_constraint_pointing_to_missing_field (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedQueryNameClashTests.test_hidden_m2m_to_m2m (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.AccessorClashTests.test_fk_to_m2m (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ReverseQueryNameClashTests.test_fk_to_m2m (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ReverseQueryNameClashTests.test_fk_to_fk (unexpected)
		--- FAIL: check_framework.test_model_checks.DuplicateDBTableTests.test_no_collision_for_proxy_models (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexesTests.test_index_with_include (unexpected)
		--- FAIL: invalid_models_tests.test_models.IndexesTests.test_pointing_to_m2m_field (unexpected)
		--- FAIL: check_framework.test_model_checks.IndexNameTests.test_collision_across_apps (unexpected)
		--- FAIL: check_framework.test_model_checks.IndexNameTests.test_collision_in_same_model (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.SelfReferentialFKClashTests.test_clash_under_explicit_related_name (unexpected)
		--- FAIL: invalid_models_tests.test_models.ShadowingFieldsTests.test_id_clash (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.RelativeFieldTests.test_many_to_many_with_useless_options (unexpected)
		--- FAIL: invalid_models_tests.test_models.JSONFieldTests.test_ordering_pointing_to_json_field_value (unexpected)
		--- FAIL: invalid_models_tests.test_models.ConstraintsTests.test_unique_constraint_pointing_to_m2m_field (unexpected)
		--- FAIL: sites_framework.tests.CurrentSiteManagerChecksTests.test_invalid_field_type (unexpected)
		--- FAIL: invalid_models_tests.test_models.OtherModelTests.test_ordering_non_iterable (unexpected)
		--- FAIL: proxy_models.tests.ProxyModelTests.test_new_fields (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedQueryNameClashTests.test_hidden_fk_to_integer (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ExplicitRelatedQueryNameClashTests.test_fk_to_m2m (unexpected)
		--- FAIL: invalid_models_tests.test_models.UniqueTogetherTests.test_pointing_to_m2m (unexpected)
		--- FAIL: invalid_models_tests.test_models.ConstraintsTests.test_unique_constraint_with_include_required_db_features (unexpected)
		--- FAIL: invalid_models_tests.test_models.ConstraintsTests.test_unique_constraint_include_pointing_to_fk (unexpected)
		--- FAIL: invalid_models_tests.test_models.OtherModelTests.test_non_valid (unexpected)
		--- FAIL: invalid_models_tests.test_models.OtherModelTests.test_ordering_pointing_to_missing_related_field (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.AccessorClashTests.test_fk_to_fk (unexpected)
		--- FAIL: invalid_models_tests.test_models.ConstraintsTests.test_unique_constraint_with_condition_required_db_features (unexpected)
		--- FAIL: invalid_models_tests.test_relative_fields.ReverseQueryNameClashTests.test_fk_to_integer (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_1) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63582 roachtest: django failed [C-test-failure O-roachtest O-robot branch-ricky-test-upgrade-roachprod release-blocker]
- #63504 roachtest: django failed [C-test-failure O-roachtest O-robot branch-63484 release-blocker]
- #63348 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.2]
- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2898497&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2898497&tab=artifacts#/django) on master @ [4dc05cceea254854317a9374a4b88df06c0946a6](https://github.com/cockroachdb/cockroach/commits/4dc05cceea254854317a9374a4b88df06c0946a6):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:204,django.go:216,test_runner.go:777: 
		Tests run on Cockroach v21.1.0-alpha.3-2478-g4dc05cceea
		Tests run against django cockroach-3.2.x
		8966 Total Tests Run
		8965 tests passed
		1 test failed
		618 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_1) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63348 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.2]
- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2907568&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2907568&tab=artifacts#/django) on master @ [66a1657247f8e1cd84ce22a32321fda6352d789f](https://github.com/cockroachdb/cockroach/commits/66a1657247f8e1cd84ce22a32321fda6352d789f):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:204,django.go:216,test_runner.go:777: 
		Tests run on Cockroach v21.1.0-alpha.3-2560-g66a1657247
		Tests run against django cockroach-3.2.x
		8970 Total Tests Run
		8969 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_1) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63348 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.2]
- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2917180&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2917180&tab=artifacts#/django) on master @ [9e821aa456bfbe8b5ec3a8ef79ae3954d52cb675](https://github.com/cockroachdb/cockroach/commits/9e821aa456bfbe8b5ec3a8ef79ae3954d52cb675):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:204,django.go:216,test_runner.go:777: 
		Tests run on Cockroach v21.1.0-alpha.3-2664-g9e821aa456
		Tests run against django cockroach-3.2.x
		8970 Total Tests Run
		8969 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_1) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2920165&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2920165&tab=artifacts#/django) on master @ [3cd29547884feb24b150b67ca2a88073798a35fd](https://github.com/cockroachdb/cockroach/commits/3cd29547884feb24b150b67ca2a88073798a35fd):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:204,django.go:216,test_runner.go:777: 
		Tests run on Cockroach v21.1.0-alpha.3-2692-g3cd2954788
		Tests run against django cockroach-3.2.x
		8970 Total Tests Run
		8969 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_1) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2921792&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2921792&tab=artifacts#/django) on master @ [2d7f9a08bac49040a6d77c5465e2b4fd8fdac1f4](https://github.com/cockroachdb/cockroach/commits/2d7f9a08bac49040a6d77c5465e2b4fd8fdac1f4):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:204,django.go:216,test_runner.go:777: 
		Tests run on Cockroach v21.1.0-alpha.3-2698-g2d7f9a08ba
		Tests run against django cockroach-3.2.x
		8970 Total Tests Run
		8969 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_1) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2924709&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2924709&tab=artifacts#/django) on master @ [2dbd33ce319439bc424a52896a03e0a7ccff07f1](https://github.com/cockroachdb/cockroach/commits/2dbd33ce319439bc424a52896a03e0a7ccff07f1):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:204,django.go:216,test_runner.go:777: 
		Tests run on Cockroach v21.1.0-alpha.3-2712-g2dbd33ce31
		Tests run against django cockroach-3.2.x
		8970 Total Tests Run
		8969 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_1) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2928832&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2928832&tab=artifacts#/django) on master @ [9f198e190812bf301e8a2126cf43cd78e1cfb163](https://github.com/cockroachdb/cockroach/commits/9f198e190812bf301e8a2126cf43cd78e1cfb163):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:777: 
		Tests run on Cockroach v21.1.0-alpha.3-2769-g9f198e1908
		Tests run against django cockroach-3.2.x
		8970 Total Tests Run
		8969 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_1) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2934234&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2934234&tab=artifacts#/django) on master @ [394543ab7bd9ff4be8acad9d1fdc892722032293](https://github.com/cockroachdb/cockroach/commits/394543ab7bd9ff4be8acad9d1fdc892722032293):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:777: 
		Tests run on Cockroach v21.1.0-alpha.3-2837-g394543ab7b
		Tests run against django cockroach-3.2.x
		8973 Total Tests Run
		8972 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_1) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2940313&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2940313&tab=artifacts#/django) on master @ [d84625c42d11e38775b6a1ff3f4089e839fe3e82](https://github.com/cockroachdb/cockroach/commits/d84625c42d11e38775b6a1ff3f4089e839fe3e82):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:777: 
		Tests run on Cockroach v21.1.0-alpha.3-2897-gd84625c42d
		Tests run against django cockroach-3.2.x
		8973 Total Tests Run
		8972 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_1) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2941631&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2941631&tab=artifacts#/django) on master @ [42e2e4265087751695afc05fc5ad8ec534fe0121](https://github.com/cockroachdb/cockroach/commits/42e2e4265087751695afc05fc5ad8ec534fe0121):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:777: 
		Tests run on Cockroach v21.1.0-alpha.3-2918-g42e2e42650
		Tests run against django cockroach-3.2.x
		8973 Total Tests Run
		8972 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_1) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2945603&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2945603&tab=artifacts#/django) on master @ [dbf711a83969210bcf706d65f53f888856da6ad0](https://github.com/cockroachdb/cockroach/commits/dbf711a83969210bcf706d65f53f888856da6ad0):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:777: 
		Tests run on Cockroach v21.2.0-alpha.00000000-146-gdbf711a839
		Tests run against django cockroach-3.2.x
		8973 Total Tests Run
		8972 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #64575 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.2]
- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2949834&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2949834&tab=artifacts#/django) on master @ [f9dca40d3afb0a605eeee76bec4a07ff65d6fec0](https://github.com/cockroachdb/cockroach/commits/f9dca40d3afb0a605eeee76bec4a07ff65d6fec0):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:777: 
		Tests run on Cockroach v21.2.0-alpha.00000000-186-gf9dca40d3a
		Tests run against django cockroach-3.2.x
		8973 Total Tests Run
		8972 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #64575 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.2]
- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2954885&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2954885&tab=artifacts#/django) on master @ [5c93801544d6ae54b369c5beebb31a298972c23e](https://github.com/cockroachdb/cockroach/commits/5c93801544d6ae54b369c5beebb31a298972c23e):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:777: 
		Tests run on Cockroach v21.2.0-alpha.00000000-260-g5c93801544
		Tests run against django cockroach-3.2.x
		8973 Total Tests Run
		8972 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2957999&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2957999&tab=artifacts#/django) on master @ [dee0558010170d938ead781d82514fd0bb3b0b88](https://github.com/cockroachdb/cockroach/commits/dee0558010170d938ead781d82514fd0bb3b0b88):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:777: 
		Tests run on Cockroach v21.2.0-alpha.00000000-288-gdee0558010
		Tests run against django cockroach-3.2.x
		8980 Total Tests Run
		8979 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2960943&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2960943&tab=artifacts#/django) on master @ [329a636debe0d968f2f4d6fe0daef957c4c41583](https://github.com/cockroachdb/cockroach/commits/329a636debe0d968f2f4d6fe0daef957c4c41583):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:777: 
		Tests run on Cockroach v21.2.0-alpha.00000000-324-g329a636deb
		Tests run against django cockroach-3.2.x
		8980 Total Tests Run
		8979 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2964420&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2964420&tab=artifacts#/django) on master @ [704e841bbc440deb4321447d9f36fc6ba41c39df](https://github.com/cockroachdb/cockroach/commits/704e841bbc440deb4321447d9f36fc6ba41c39df):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:777: 
		Tests run on Cockroach v21.2.0-alpha.00000000-336-g704e841bbc
		Tests run against django cockroach-3.2.x
		8980 Total Tests Run
		8979 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
- #62295 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-20.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2965590&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2965590&tab=artifacts#/django) on master @ [ac033403c8fbb699cff200316e432dc60b47be6e](https://github.com/cockroachdb/cockroach/commits/ac033403c8fbb699cff200316e432dc60b47be6e):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:777: 
		Tests run on Cockroach v21.2.0-alpha.00000000-354-gac033403c8
		Tests run against django cockroach-3.2.x
		8980 Total Tests Run
		8979 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2969630&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2969630&tab=artifacts#/django) on master @ [4da45c4e6b4ba223b832eb6a1f675addbc479f76](https://github.com/cockroachdb/cockroach/commits/4da45c4e6b4ba223b832eb6a1f675addbc479f76):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:784: 
		Tests run on Cockroach v21.2.0-alpha.00000000-393-g4da45c4e6b
		Tests run against django cockroach-3.2.x
		8980 Total Tests Run
		8979 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2973749&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2973749&tab=artifacts#/django) on master @ [88754f2866fde1f37620b04c6a708eefc0b5be54](https://github.com/cockroachdb/cockroach/commits/88754f2866fde1f37620b04c6a708eefc0b5be54):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:784: 
		Tests run on Cockroach v21.2.0-alpha.00000000-434-g88754f2866
		Tests run against django cockroach-3.2.x
		8980 Total Tests Run
		8979 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2978383&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2978383&tab=artifacts#/django) on master @ [5c0b2ec40f06678c0a09c48bcca2904354765734](https://github.com/cockroachdb/cockroach/commits/5c0b2ec40f06678c0a09c48bcca2904354765734):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:784: 
		Tests run on Cockroach v21.2.0-alpha.00000000-472-g5c0b2ec40f-dirty
		Tests run against django cockroach-3.2.x
		8980 Total Tests Run
		8979 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2982472&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2982472&tab=artifacts#/django) on master @ [68b69b6e24c4016e724492b19a0567ad189fd8fd](https://github.com/cockroachdb/cockroach/commits/68b69b6e24c4016e724492b19a0567ad189fd8fd):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:784: 
		Tests run on Cockroach v21.2.0-alpha.00000000-500-g68b69b6e24
		Tests run against django cockroach-3.2.x
		8980 Total Tests Run
		8979 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2985381&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2985381&tab=artifacts#/django) on master @ [b706c913c07890d04d6bbec4d9d7cd8fea609b95](https://github.com/cockroachdb/cockroach/commits/b706c913c07890d04d6bbec4d9d7cd8fea609b95):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:784: 
		Tests run on Cockroach v21.2.0-alpha.00000000-520-gb706c913c0
		Tests run against django cockroach-3.2.x
		8980 Total Tests Run
		8979 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2988804&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2988804&tab=artifacts#/django) on master @ [25e26a5f0361fcfbf71cee86f963b93f806604d8](https://github.com/cockroachdb/cockroach/commits/25e26a5f0361fcfbf71cee86f963b93f806604d8):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:784: 
		Tests run on Cockroach v21.2.0-alpha.00000000-542-g25e26a5f03
		Tests run against django cockroach-3.2.x
		8980 Total Tests Run
		8979 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2993935&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2993935&tab=artifacts#/django) on master @ [eecead5c3aa93b5718a57c47970d67ee3035e794](https://github.com/cockroachdb/cockroach/commits/eecead5c3aa93b5718a57c47970d67ee3035e794):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:792: 
		Tests run on Cockroach v21.2.0-alpha.00000000-579-geecead5c3a
		Tests run against django cockroach-3.2.x
		8986 Total Tests Run
		8985 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2998287&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2998287&tab=artifacts#/django) on master @ [c305dbcff1bcf621ae8edf058b220033c98d6526](https://github.com/cockroachdb/cockroach/commits/c305dbcff1bcf621ae8edf058b220033c98d6526):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:792: 
		Tests run on Cockroach v21.2.0-alpha.00000000-626-gc305dbcff1
		Tests run against django cockroach-3.2.x
		8986 Total Tests Run
		8985 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3001941&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3001941&tab=artifacts#/django) on master @ [bf71898e043692cc87d2c232fe3066054275ec29](https://github.com/cockroachdb/cockroach/commits/bf71898e043692cc87d2c232fe3066054275ec29):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:245,orm_helpers.go:171,django.go:206,django.go:218,test_runner.go:792: 
		Tests run on Cockroach v21.2.0-alpha.00000000-651-gbf71898e04
		Tests run against django cockroach-3.2.x
		8986 Total Tests Run
		8985 tests passed
		1 test failed
		619 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: schema.tests.SchemaTests.test_ci_cs_db_collation (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist21_2) is available in the artifacts' django log
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
# From https://go.crdb.dev/p/roachstress, perhaps edited lightly.
caffeinate ./roachstress.sh django
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #63345 roachtest: django failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/django.go
+++ b/pkg/cmd/roachtest/django.go
@@ -20,7 +20,7 @@ var djangoReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)(
 var djangoCockroachDBReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d+)$`)
 
 var djangoSupportedTag = "cockroach-3.2.x"
-var djangoCockroachDBSupportedTag = "3.2"
+var djangoCockroachDBSupportedTag = "3.2.1"
 
 func registerDjango(r *testRegistry) {
 	runDjango := func(

